cmd.go

```diff
@@ -87,7 +88,26 @@ func (r *Runner) Setup(cmd *cobra.Command, runner interface{}, cfg interface{})
 		Short: "output the config environment variables and exits",
 		Args:  cobra.NoArgs,
 		RunE: func(cmd *cobra.Command, args []string) error {
-			vars, err := GetVariablesFromConfig(r.prefix, cfg)
+			var cfgMgr config.Provider
+			app := fx.New(
+				Module,
+				runner,
+				fx.Populate(&cfgMgr),
+				fx.Invoke(
```

Comment:
I would move this above `runner`. We should test to ensure the `runner`'s lifecycles don't get executed (such as `server.Lifecycle`).

Reply:
Will look into the testing bits for init.